react/components/atoms/button-tab/button-tab.js

```diff
@@ -36,6 +36,7 @@ export class ButtonTab extends mix(PureComponent).with(IdentifiableMixin) {
             colorSelected: PropTypes.string,
             selected: PropTypes.bool,
             disabled: PropTypes.bool,
+            fill: PropTypes.bool,

```

Comment:
Shouldn't this be a string instead, so you could specify the fill value? You'd need the same approach as color: color + colorSelected.

Reply:
We normally don't use different fill and stroke colors, so I was trying to simplify the logic

Reply:
Indeed, but since this is a low-level component it should allow that. Example: https://github.com/ripe-tech/ripe-components-vue/blob/master/vue/components/ui/atoms/icon/icon.vue#L30